slice-core-api/src/main/java/com/cognifide/slice/api/injector/InjectorConfig.java

```diff
@@ -82,4 +84,12 @@ public String getBundleNameFilter() {
 	public String getApplicationPath() {
 		return applicationPath;
 	}
+
+	public InjectorCreationFailListener getListener() {
+		return listener;
+	}
+
+	public void setListener(InjectorCreationFailListener listener) {
```

Comment:
Maybe we can pass this listener as a constructor parameter or expose it in InjectorRunner getter? Or maybe the whole InjectorRunner should implement this interface?